Src/Screens/ProfileScreen/Screens/MyPayment/MyPayment.swift

```diff
@@ -76,7 +76,7 @@ extension MyPayment: Presentable {
                 viewController.present(directDebitSetup, options: [.autoPop])
             }
 
-            if result.data?.isSwitchingBankAccount() ?? false {
+            if result.data?.directDebitStatus == .pending {
```

Comment:
i think here should be needs_setup

Reply:
@meletios this is for hiding the "Connect autogiro"-button, which is only to be hidden if it's pending, right?

Reply:
We need to hide it, when its pending or connected, eller?

Reply:
only pending :D

Reply:
yeap correct. I thought we are talking about the banner :laughing:

Reply:
hehe no